list_domains/dune

```diff
@@ -0,0 +1,7 @@
+(executable
+  (name list_domains)
+  (public_name list_domains)
+  (package xapi-xenopsd-xc)
+  (flags (:standard -bin-annot))
```

Comment:
I don't think we need `bin-annot` here specifically, `merlin` works fairly well with the defaults.

Reply:
I'll remove that line.